lint.sh

```diff
@@ -73,7 +72,7 @@ check_bash(){
 check_python(){
   while read script
   do
-    python -m py_compile  $script \
+    flake8 $script \
       && echo "Python syntax ok: $script" \
       || { echo "Python syntax fail $script"; rc=1; }
   done < <(find ${fargs[@]} -iname \*.py)
```

Comment:
Any downside to just running flake8 on the entire dir, rather than on each found py script?